src/geometry/cgal/cgalutils-applyops.cc

```diff
@@ -149,293 +149,9 @@ shared_ptr<const Geometry> applyOperator3D(const Geometry::Geometries& children,
     std::string opstr = op == OpenSCADOperator::INTERSECTION ? "intersection" : op == OpenSCADOperator::DIFFERENCE ? "difference" : op == OpenSCADOperator::UNION ? "union" : "UNKNOWN";
     LOG(message_group::Error, "exception in CGALUtils::applyBinaryOperator %1$s: %2$s", opstr, e.what());
   }
-  return shared_ptr<Geometry>(N);
+  return N;
 }
 
-bool applyHull(const Geometry::Geometries& children, PolySet& result)
```

Comment:
wow, is this `applyHull` never being used?

Reply:
It was moved to boolean_utils earlier, and I forgot to remove it